src/cljc/rems/common/catalogue_util.cljc

```diff
@@ -61,6 +73,20 @@
                                                                                                           :ega-organization "https:/ega.org/"}))
         "setting custom EGA organization works"))
 
+  (testing "DOI"
+    (is (= nil (catalogue-item-more-info-url {:resource/ext-id "10.1109/5.771073"} nil {})))
```

Comment:
We could clarify here as well that without the feature flag it should not match?